website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,211 @@ Learn more about Karpenter and how to get started below.
+
+This guide will also assume you have the `aws` and `kubectl` CLI tools installed.
+You can also perform many of these steps in the console, but we will use the command line for simplicity.
+
+## Get a cluster
+To use Karpenter, you must be running a supported Kubernetes cluster on a supported cloud provider.
+Currently, only EKS on AWS is supported.
+Recommended ways of deploying EKS on AWS to use with Karpenter include:
+
+* [Creating an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html): Use `eksctl` or `aws` CLI tools or the AWS Management console to deploy an EKS cluster.
+* [Amazon EKS Blueprints for Terraform](https://aws-ia.github.io/terraform-aws-eks-blueprints): Follow a basic [Getting Started](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/getting-started/) guide and also add modules and add-ons. This includes a [Karpenter](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/add-ons/karpenter/) add-on that lets you bypass the instructions in this guide for setting up Karpenter.
+
+Although not supported, Karpenter could work on other Kubernetes distributions running on AWS. For example:
+* [kOps](https://kops.sigs.k8s.io/operations/karpenter/): These instructions describe how to create a kOps Kubernetes cluster in AWS that includes Karpenter.
+
+If you have not already configured Karpenter when you set up your Kubernetes cluster in AWS, follow the instructions below to add Karpenter to your existing cluster.
+
+## Add Karpenter to an existing cluster
+
+To add Karpenter to your cluster, you need to create IAM roles, add tags to subnets and security groups, and update the aws-auth ConfigMap.
+
+### Create IAM role
+
+To get started, create a new IAM role for the Karpenter controller.
+
+The controller will be using [IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) which requires an OIDC endpoint.
+
+If you have another option for using IAM credentials with workloads (e.g. [kube2iam](https://github.com/jtblin/kube2iam)) your steps will be different.
+
+Set a variable for your cluster name.
+
+```bash
+CLUSTER_NAME=<your cluster name>
+```
+
+Set other variables from your cluster configuration.
+
+{{% script file="./scripts/step04-env.sh" language="bash" %}}
+
+Use that information to create our IAM role, inline policy, and trust relationship.
+
+{{% script file="./scripts/step05-controller-iam.sh" language="bash" %}}
+
+### Add tags to subnets and security groups
+
+Next add tags to your nodegroup subnets so Karpenter will know which subnets to use.
+
+{{% script file="./scripts/step06-tag-subnets.sh" language="bash" %}}
+
+Add tags to the security groups.
+This command only tags the security groups for the first nodegroup in the cluster.
+If you have multiple nodegroups or multiple security groups you will need to decide which one Karpenter should use.
```

Comment:
The wording seems misleading here to me. From reading this, I would assume that Karpenter isn't going to be a groupless node autoscaler. Maybe we can do 
```suggestion
This command only tags the security groups used by the first nodegroup in the cluster.
If you have multiple nodegroups or multiple security groups you will need to decide which one Karpenter should use.
```